cmake/projects/tcl/scripts/android_def.h

```diff
@@ -0,0 +1,3 @@
+#if __ANDROID_API__ < 21
+#undef HAVE_STRUCT_DIRENT64
+#endif
```

Comment:
http://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file

Reply:
fixed